### PR TITLE
crashplan: fix .out in LD_LIBRARY_PATH

### PR DESCRIPTION
**Pull request by jerith666**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/backup/crashplan/default.nix [hunk 1]
```diff
@@ -81,6 +81,6 @@ in stdenv.mkDerivation rec {
       --replace crashplan/skin skin \
       --replace bin/CrashPlanDesktop CrashPlanDesktop
 
-    wrapProgram $out/bin/CrashPlanDesktop --prefix LD_LIBRARY_PATH ":" "${gtk2}/lib:${glib}/lib:${libXtst}/lib"
+    wrapProgram $out/bin/CrashPlanDesktop --prefix LD_LIBRARY_PATH ":" "${gtk2.out}/lib:${glib.out}/lib:${libXtst.out}/lib"
```
Comment (joachifm) [inline pkgs/applications/backup/crashplan/default.nix]: This is perhaps more convenient as `makeLibaryPath`

**Comment by jagajaga** [comment]
cc @vcunat

**Comment by joachifm** [comment]
Pushed to master. Thank you.

**Comment by jerith666** [comment]
Thanks, sorry for not getting to the `makeLibraryPath` cleanup more quickly.

**Pull request closed without merging**